### ISSUE TITLE
why ws4w can't use Ethernet to share network

### ISSUE THREAD

**Issue by RogerSugit** [micahmo/WgServerforWindows#74]
My pc have wifi and ethernet. I want to enable ICS for ethernet, but ICS is allways be enabled for wifi.
Is it my system's issue?

**Comment by micahmo** [comment]
Hi @RogerSugit, when you select the option to Enable Internet Sharing, you are presented with a list of all of your network adapters, and you can choose the one you want to share. Are you saying that you pick the ethernet adapter from the list, but the Wi-Fi adapter's connection is the one that's shared?

If so, you may need to look in Windows. Open Network Connections in the Control Panel. Right-click > Properties on the Wi-Fi adapter, go to the Sharing tab, and ensure that "Allow other network users to connect through this computer's Internet connection" is unchecked. Then go to the ethernet adapter and ensure the same option is checked. Unfortunately I've found sometimes the only way to enable this is through Windows.

**Comment by RogerSugit** [comment]
Yes, I picked the ethernet adapter to share, but the Wi-Fi adapter's connection is the one that's shared.
The solution you said I have tried for many times, but it didn't work, ICS is always working on WiFi adapter.

**Comment by RogerSugit** [comment]
> 

Yes, I picked the ethernet adapter to share, but the Wi-Fi adapter's connection is the one that's shared.
The solution you said I have tried for many times, but it didn't work, ICS is always working on WiFi adapter.

**Comment by RogerSugit** [comment]
Yes, I picked the ethernet adapter to share, but the Wi-Fi adapter's connection is the one that's shared.
The solution you said I have tried for many times, but it didn't work, ICS is always working on WiFi adapter.

**Comment by micahmo** [comment]
Hi @RogerSugit, are you still experiencing this issue? I was never able to reproduce. Just want to know if you want me to look into this further or if you have moved on to something else.
